pkg/component/coredns/coredns_test.go

```diff
@@ -808,7 +801,7 @@ status: {}
 			})
 		})
 
-		Context("kubernetes version < 1.21.0, w/ apiserver host, w/ pod annotations", func() {
+		Context("kubernetes version < 1.23.0, w/ apiserver host, w/ pod annotations", func() {
```

Comment:
Please update the kubernetes version 7 lines below.

Reply:
Already addressed in https://github.com/gardener/gardener/pull/8087/commits/ae2e81dcc1e53869385e46e94962a27e91e778da